src/main/java/nl/tudelft/ti2206/group9/audio/AudioPlayer.java

```diff
@@ -69,6 +69,27 @@ public final void play() {
 	}
 
 	/**
+	 * Check whether an AudioPlayer is already running or not,
+	 * and if not, also starts the AudioPlayer.
+	 */
+	public final void playCheck() {
+		if (!this.isRunning()) {
+			this.play();
+		}
+
+	}
+
+	/**
+	 * Check whether an AudioPlayer is already running or not,
+	 * and if so, also stop the AudioPlayer.
+	 */
+	public final void stopCheck() {
+		if (this.isRunning()) {
+			this.stop();
+		}
+	}
```

Comment:
Why are the playCheck and stopCheck not integrated in play and stop themselves? Are the play() and stop() method used separately somewhere? I'm thinking you could write something like:

    public final void play() {
        if(this.isRunning) return;
        try {...}
    }

<logic,organization,review